### PR TITLE
import: SUDOC

### LINKED ISSUE #3363: Add import from SUDOC by SRU
### Missing feature
SUDOC now exports SRU data. Add the ability to import documents from SUDOC and converting UNIMARC to json same as for BNF.

### PR DESCRIPTION
**Pull request by rerowep**
* Closes #3363.

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

### CODE REVIEW

File: rero_ils/modules/imports/views.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def get(self, **kwargs):
             where = 'anywhere'
             relation = 'all'
             what = query
+        print('---->', query, where, relation, what)
```
Comment (lauren-d) [inline rero_ils/modules/imports/views.py:91]: seems useless

**Review by lauren-d — COMMENTED** [review]
just for memo: BNF unimarc mapping needs to be tested and approved for SUDOC

**Review by zannkukai — APPROVED** [review]
Seems ok for me but I'm not specialist.

**Comment by rerowep** [comment]
Correct: BNF unimarc mapping needs to be tested and approved for SUDOC.
I tried marc21 but SUDOC don't gives results for this format even we can ask for it.

**Review by jma — APPROVED** [review]

**Review by lauren-d — APPROVED** [review]

**Review by zannkukai — APPROVED** [review]

**Comment by PascalRepond** [comment]
## PO Tests

- [ ] Rep: Search by ISBN/EAN doesn't yield results. Example : 9782364804647 ([https://www.sudoc.fr/](https://www.sudoc.fr/245428240)[245428240](https://www.sudoc.fr/245428240))

**Review by PascalRepond — APPROVED** [review]

**Comment by Ewa666** [comment]
image

Sudoc has a lot of "lc" in 6XX (imported from OCLC). It is blocked when importing records.

**Comment by PascalRepond** [comment]
> image
> 
> Sudoc has a lot of "lc" in 6XX (imported from OCLC). It is blocked when importing records.

Thank you for test, please use existing comment above to add remarks.

**Pull request merged**